01-path-basics/103-kubernetes-concepts/readme.adoc

```diff
@@ -214,9 +214,12 @@ Get the list of pod:
 Verify that the pod came up fine (ensure nothing else is running on port 8088):
 
 	$ kubectl -n default port-forward $(kubectl -n default get pod -l name=nginx-pod -o jsonpath='{.items[0].metadata.name}') 8088:80
+
+From a new terminal window run:
+
 	$ open http://localhost:8088/
 
-If you are running a Linux distribution that doesn't support `open`, try `xdg-open`, `gnome-open`, or `gvfs-open` instead.
+If you are running a Linux distribution that doesn't support `open`, try `xdg-open`, `gnome-open`, or `gvfs-open` instead.  From an AWS Cloud9 environment, use curl instead.
 
 This opens up a browser window and shows the NGINX main page:
```

Comment:
Remove this line as well to avoid confusion

This opens up a browser window and shows the NGINX main page: